src/components/Inputs/CorInputCheckbox/CorInputCheckbox.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../variables';
+
+.cor_input_checkbox {
+    margin-bottom: 12px;
+    position: relative;
+
+    input[type='checkbox'] {
+        appearance: none;
+        background: white;
+        border: solid 2px black;
+        border-radius: 5px;
+        display: grid;
+        height: 20px;
+        margin: 0;
+        place-content: center;
+        width: 20px;
+        z-index: 1;
+
+        &:disabled {
+            background: $grey-8;
+            border: solid 1px $grey-6;
+        }
+    }
+
+    input[type='checkbox'] + svg {
+        left: 0;
+        position: absolute;
+        transform: scale(0);
```

Comment:
transform don't have any effect

Reply:
It does. It's for the animation to start with a scale of 0.